### PR TITLE
PARETO - Sphinx documentation

### PR DESCRIPTION
**Pull request by MAZamarripa**
## Fixes/Addresses/Summary/Motivation: adding PARETO documentation (index, getting started, model libraries, and utilities). 

(Replace this with the issue # fixed or addressed in this PR as well as any other related PRs.  If
no issue or PRs exist, then add a brief statement on what changes this PR is bringing in.)

## Changes proposed in this PR:
- adding rst documentation files
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    utilities/index
+    faq
+    license
+    copyright
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```
Comment (melody-shellman) [inline docs/index.rst:74]: I think the reference to 'modindex' is broken. When I build the files and click on this link, I get a "file couldn't be accessed" error

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+* :ref:`search`
```
Comment (melody-shellman) [inline docs/index.rst:75]: The 'search' reference opens up to an empty page for me.  Perhaps we remove this reference until we build out the search page?

**Review by melody-shellman — CHANGES_REQUESTED** [review]
A few minor changes needed for broken links. Otherwise looks AWESOME. I'm so excited to update the RST files instead of word !!

File: docs/index.rst (hunk 1 above)
Comment (MAZamarripa) [reply docs/index.rst]: removed

File: docs/index.rst (hunk 2 above)
Comment (MAZamarripa) [reply docs/index.rst]: removed

**Review by Andresj89 — DISMISSED** [review]
LGTM

**Comment by NienkeWagenaar** [comment]
Looks good to me. I got some warnings which I fixed, other than that I think it's good to go.

**Review by NienkeWagenaar — APPROVED** [review]

**Review by melody-shellman — APPROVED** [review]
Changes look good !

**Pull request merged**